cmd/state-offline-uninstaller/uninstall.go

```diff
@@ -67,26 +117,28 @@ func (r *runner) Run(params *Params) error {
 			"Directory does not look like an install directory, are you sure you want to proceed?",
 			p.BoolP(true))
 		if err != nil {
-			return errs.Wrap(err, "Error getting confirmation for installing")
+			return r.handleFailure(err, "Error getting confirmation for installing", installerDimensions)
 		}
 
 		if !confirmUninstall {
-			return errs.New("ActiveState license not found in uninstall directory. Please specify a valid uninstall directory.")
+			return r.handleFailureNewErr("ActiveState license not found in uninstall directory. Please specify a valid uninstall directory.", installerDimensions)
 		}
 	}
 
 	r.out.Print("Removing environment configuration")
 	err = r.removeEnvPaths()
 	if err != nil {
-		return errs.Wrap(err, "Error removing environment path")
+		return r.handleFailure(err, "Error removing environment path", installerDimensions)
 	}
 
 	r.out.Print("Removing installation directory")
 	err = os.RemoveAll(params.path)
 	if err != nil {
-		return errs.Wrap(err, "Error removing installation directory")
+		return r.handleFailure(err, "Error removing installation directory", installerDimensions)
 	}
 
+	r.analytics.Event(ac.CatRuntimeUsage, "success", installerDimensions)
```

Comment:
"Success" is not an action of CatRuntimeUsage. See https://docs.google.com/document/d/12-IHGV6iBGE12p61cwKvc16YzahplKxJYSh30HKaBRk/edit#heading=h.u1lsd920bwhf

Reply:
Offline installer now uses its own category